.github/workflows/rust.yml

```diff
@@ -32,6 +32,6 @@ jobs:
     - name: Rustfmt
       run: cargo fmt --all --check
     - name: Verify regenerated files
-      run: ./scripts/unicode.py && diff tables.rs src/tables.rs
+      run: ./scripts/unicode.py && rustfmt tables.rs && diff tables.rs src/tables.rs
```

Comment:
issue: no, let's just `#[rustfmt::skip]` the module so people don't have to remember to do this.

Reply:
Should we also start `skip`ping testdata.rs?

Reply:
sure yeah